lib/login.ts

```diff
@@ -144,19 +144,21 @@ export type Callback<TResult> = (error?: Error, result?: TResult) => void;
  * Provides a UserTokenCredentials object and the list of subscriptions associated with that userId across all the applicable tenants.
  * This method is applicable only for organizational ids that are not 2FA enabled otherwise please use interactive login.
  *
+ * For personal accounts, credentials created without passing a domain won't be able to access most of the account resources automatically. In this case you will need to call `buildTenantList` to gather the list of tenants so that the ID of one of them can be passed into the `setDomain` method of the returned credential. Once the domain is set, you will be able to access resources from subscriptions in that tenant.
+ *
```

Comment:
As mentioned in https://github.com/Azure/ms-rest-nodeauth/pull/91#discussion_r437629333, the readme talks about how to use buildTenantList, we do not need to have that information in docs for every auth method.

```suggestion
 * Provides a UserTokenCredentials object and the list of subscriptions associated with that userId across all the applicable tenants. 
 * This list may be empty and the credentials may generate invalid tokens when using personal accounts if `domain` is not set to the tenant id in the `options`.
 * This method is applicable only for organizational ids that are not 2FA enabled otherwise please use interactive login.
 *
```

Reply:
I believe that I didn't see the original suggestion from a web cache issue. I usually try to follow up, even with a comment or a reaction.

I don't think this is correct `may generate invalid tokens` though. I'll post my proposal in a bit.